libraries/Adapters/Microsoft.Bot.Builder.Adapters.Twilio/TwilioAdapter.cs

```diff
@@ -46,8 +47,9 @@ public TwilioAdapter(ITwilioAdapterOptions options)
             }
 
             _options = options;
+            _twilioApi = twilioApi;
```

Comment:
I think that we should validate that `ITwilioClient` must be not null.

Reply:
Keep in mind that if we add a new validation, we have to create a new set of test